Tests/XCUITests/NavigationTest.swift

```diff
@@ -194,83 +195,101 @@ class NavigationTest: BaseTestCase {
-            XCTAssertTrue(app.menuItems["Look Up"].exists)
+            navigator.performAction(Action.CloseURLBarOpen)
+            navigator.nowAt(NewTabScreen)
+            navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
+            longPressLinkOptions(optionSelected: "Copy Link")
+            navigator.goto(NewTabScreen)
+            waitForExistence(app.textFields["url"])
+            app.textFields["url"].press(forDuration: 2)
+
+            app.tables.otherElements[ImageIdentifiers.paste].tap()
+            app.buttons["Go"].tap()
+            waitUntilPageLoad()
+            waitForValueContains(app.textFields["url"], value: website_2["moreLinkLongPressInfo"]!)
         }
+    }
 
-        app.textFields["address"].typeText("\n")
-        waitUntilPageLoad()
-        waitForNoExistence(app.staticTexts["XCUITests-Runner pasted from Fennec"])
-
-        app.textFields["url"].press(forDuration: 3)
-        app.tables.otherElements[ImageIdentifiers.copyLink].tap()
-
-        sleep(2)
-        app.textFields["url"].tap()
-        // Since the textField value appears all selected first time is clicked
-        // this workaround is necessary
-        waitForNoExistence(app.staticTexts["XCUITests-Runner pasted from Fennec"])
-        app.textFields["address"].tap()
-        waitForExistence(app.menuItems["Copy"])
-        if iPad() {
-            XCTAssertTrue(app.menuItems["Copy"].exists)
-            XCTAssertTrue(app.menuItems["Cut"].exists)
-            XCTAssertTrue(app.menuItems["Look Up"].exists)
-            XCTAssertTrue(app.menuItems["Share…"].exists)
-            XCTAssertTrue(app.menuItems["Paste & Go"].exists)
-            XCTAssertTrue(app.menuItems["Paste"].exists)
+    func testLongPressOnAddressBar() throws {
+        if processIsTranslatedStr() == m1Rosetta {
+            throw XCTSkip("Copy & paste may not work on M1")
         } else {
-            XCTAssertTrue(app.menuItems["Copy"].exists)
-            XCTAssertTrue(app.menuItems["Cut"].exists)
-            XCTAssertTrue(app.menuItems["Look Up"].exists)
+            // This test is for populated clipboard only so we need to make sure there's something in Pasteboard
+            app.textFields["address"].typeText("www.google.com")
+            // Tapping two times when the text is not selected will reveal the menu
+            app.textFields["address"].tap()
+            waitForExistence(app.textFields["address"])
+            app.textFields["address"].tap()
+            waitForExistence(app.menuItems["Select All"])
+            XCTAssertTrue(app.menuItems["Select All"].exists)
+            XCTAssertTrue(app.menuItems["Select"].exists)
+
+            // Tap on Select All option and make sure Copy, Cut, Paste, and Look Up are shown
+            app.menuItems["Select All"].tap()
+            waitForExistence(app.menuItems["Copy"])
+            if iPad() {
+                XCTAssertTrue(app.menuItems["Copy"].exists)
```

Comment:
@isabelrios Could you please kindly confirm these menu items on your end? I cannot verify this part on my M1.

Reply:
This is what I see
![Simulator Screen Shot - iPad Air (4th generation) - 2022-11-28 at 19 33 45](https://user-images.githubusercontent.com/1897507/204354136-1dd5a737-016c-448d-b7b4-9d5db959726f.png)

Reply:
@isabelrios Thanks. How about on the iPhone? ☎️

Reply:
![Simulator Screen Shot - iPhone 14 - 2022-11-28 at 21 22 11](https://user-images.githubusercontent.com/1897507/204373591-4b04cd23-256a-4631-99ba-2056e73bbf49.png)
![Simulator Screen Shot - iPhone 14 - 2022-11-28 at 21 22 16](https://user-images.githubusercontent.com/1897507/204373649-a3d64bed-b50c-4dfd-9e25-e4de29938653.png)